code/handlers/permissions.q

```diff
@@ -158,17 +158,24 @@ exe:{v:$[(100<abs type first x);val;valp]x;
 
 qexe:{v:val x; if[maxsize<-22!v; 'err[`size][]]; v}
 
+isvar:{[f]
+  / if first element is not a symbol, not a var ref
+  if[-11h<>type f;:0b];
+  / if first element is symbol, check if it's a variable (type<100h) or named function
+  :$[99h<type get f;0b;1b];
```

Comment:
Use `not` instead of conditional?
```suggestion
  :not 99h<type get f;
```
Possible short enough to be one liner after that.